tmt-app/src/App.js

```diff
@@ -13,7 +15,15 @@ const router = createBrowserRouter([
   },
   {
     path: "/guide",
-    element: <CareerGuide />,
+    element: <ResultsPage />,
+  },
+  {
+    path: "/majorpage",
```

Comment:
`/majorpage` seems like a pretty odd url to see, should we have something more readable such as `majors` or `major` depending on what this page will display, also is this in scope?

Reply:
Yup, I will change the url. What do you mean by in scope?

Reply:
Was it related to this work